bin/commands/init/mvs/index.sh

```diff
@@ -59,56 +81,53 @@ while read -r line; do
       # warning
       print_message "Warning ZWEL0301W: ${ds} already exists and will not be overwritten. For upgrades, you must use --allow-overwrite."
     fi
-  else
-    print_message "Creating ${ds}"
-    create_data_set "${ds}" "${spec}"
-    if [ $? -ne 0 ]; then
-      print_error_and_exit "Error ZWEL0111E: Command aborts with error." "" 111
-    fi
   fi
 done <<EOF
 $(echo "${cust_ds_list}")
 EOF
+
 print_message
 
 if [ "${ds_existence}" = "true" ] &&  [ "${ZWE_CLI_PARAMETER_ALLOW_OVERWRITE}" != "true" ]; then
   print_message "Skipped writing to ${ds}. To write, you must use --allow-overwrite."
+  print_level2_message "Zowe custom data sets initialized with errors."
 else
-  ###############################
-  # copy sample lib members
-  parmlib=$(read_yaml "${ZWE_CLI_PARAMETER_CONFIG}" ".zowe.setup.dataset.parmlib")
-  for ds in ZWESIP00; do
-    print_message "Copy ${prefix}.${ZWE_PRIVATE_DS_SZWESAMP}(${ds}) to ${parmlib}(${ds})"
-    data_set_copy_to_data_set "${prefix}" "${prefix}.${ZWE_PRIVATE_DS_SZWESAMP}(${ds})" "${parmlib}(${ds})" "${ZWE_CLI_PARAMETER_ALLOW_OVERWRITE}"
```

Comment:
`data_set_copy_to_data_set` is not properly working due to the following [bug](https://github.com/zowe/zowe-install-packaging/blob/8bbb8bcbe8942578c39513e011b92eed5b1a540a/bin/libs/zos-dataset.sh#L108C5-L108C65).
The dataset name is prefixed with `//` and then again in `is_data_set_exists` resulting to `cat ////ds_name` never returning `true`. This PR is removing all occurrences of `data_set_copy_to_data_set`, which is great opportunity to fix the bug.

There is the same bug on [another line](https://github.com/zowe/zowe-install-packaging/blob/8bbb8bcbe8942578c39513e011b92eed5b1a540a/bin/libs/zos-dataset.sh#L78). Fixing this bug in not that easy, it is causing more problems.

But anyway, please, fix the **`data_set_copy_to_data_set`**